doc/py_tutorials/py_imgproc/py_thresholding/py_thresholding.markdown

```diff
@@ -4,33 +4,33 @@ Image Thresholding {#tutorial_py_thresholding}
 Goal
 ----
 
--   In this tutorial, you will learn Simple thresholding, Adaptive thresholding, Otsu's thresholding
-    etc.
--   You will learn these functions : **cv.threshold**, **cv.adaptiveThreshold** etc.
+-   In this tutorial, you will learn Simple thresholding, Adaptive thresholding and Otsu's thresholding.
+-   You will learn the functions **cv.threshold** and **cv.adaptiveThreshold**.
 
 Simple Thresholding
 -------------------
 
-Here, the matter is straight forward. If pixel value is greater than a threshold value, it is
-assigned one value (may be white), else it is assigned another value (may be black). The function
-used is **cv.threshold**. First argument is the source image, which **should be a grayscale
-image**. Second argument is the threshold value which is used to classify the pixel values. Third
-argument is the maxVal which represents the value to be given if pixel value is more than (sometimes
-less than) the threshold value. OpenCV provides different styles of thresholding and it is decided
-by the fourth parameter of the function. Different types are:
+Here, the matter is straight forward. For every pixel, the same threshold value is applied.
+If the pixel value is smaller than the threshold, it is set to 0, otherwise it is set to a maximum value.
+The function **cv.threshold** is used to apply the thresholding.
+The first argument is the source image, which **should be a grayscale image**.
+The second argument is the threshold value which is used to classify the pixel values.
+The third argument is the maximum value which is assigned to pixel values exceeding the threshold.
+OpenCV provides different types of thresholding which is given by the fourth parameter of the function.
+Basic thresholding as described above is done by using the type cv.THRESH_BINARY.
+Other thresholding types are:
 
--   cv.THRESH_BINARY
```

Comment:
I believe it is better to keep full list here, and change wording above. something like "Threshold types are:"